src/Types.jl

```diff
@@ -930,6 +930,7 @@ function ensure_resolved(ctx::Context, manifest::Manifest,
     unresolved_uuids = Dict{String,Vector{UUID}}()
     for pkg in pkgs
         has_uuid(pkg) && continue
+        !has_name(pkg) && !has_uuid(pkg) && pkgerror("Package $pkg has neither name nor uuid")
```

Comment:
Do we need `!has_uuid(pkg)` here? It looks like we check `has_uuid(pkg)` on the previous line.

Reply:
🤦